packages/loot-core/src/server/accounts/sync.js

```diff
@@ -206,13 +206,17 @@ async function downloadNordigenTransactions(
     }
 
     const {
-      transactions: { booked },
+      transactions: { booked, pending, all },
```

Comment:
If the change to line 215 is suggested then it's unnecessary to extract it here either. It should make it easier to remove `booked` and `pending` from the server's response in the future since it shouldn't require a change to the client at all then.
```suggestion
      transactions: { all },
```